packages/eslint-config-rbflabs/.prettierrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  printWidth: 100,
```

Comment:
I can see `printWidth` and `bracketSpacing` have changed compared to the original file. I think we should discuss these settings since we're setting code standards for all our projects.

Reply:
Absolutely. Do you have any strong opinions/preferences? I would like to keep the `bracketSpacing`, printWidth I don't mind at all to change.

Reply:
I'm quite used to having `bracketSpacing` set to `false` from the previous projects. But it's just my preference and I'm open to setting it to `true` (Prettier default) if you think the code is more readable.

When it comes to `printWidth`, I would increase it at least to `120`. I think it's a good balance between readability and having enough space.